packages/adapter-cloudflare-workers/README.md

```diff
@@ -14,16 +16,51 @@ Generate this file using `wrangler` from your project directory
 $ wrangler init --site my-site-name
 ```
 
-Then configure your sites build directory in the config file:
+Add the adapter to your `svelte.config.js`:
+
+```js
+import adapter from '@sveltejs/adapter-cloudflare-workers';
+const config = {
+    kit: {
+        target: '#svelte',
+        adapter: adapter()
+    }
+};
+...
```

Comment:
this seems semi-silly to me. the only thing that would be here is `export default config`. It doesn't really save any lines to replace it with `...` and is less clear. But I think you could just remove this line all together if you put `export default` above